addons/udes_stock/models/stock_location_route.py

```diff
@@ -6,3 +6,13 @@ class Route(models.Model):
 
     # Disable translation instead of renaming.
     name = fields.Char(translate=False)
+
+    def write(self, values):
+        """
+        When in a route active field is changed, set the context with active_test = False which will
+        ensure that will search even the inactive records and activate/inactivate them where needed.
+        """
+
+        if "active" in values and "active_test" not in self._context:
+            self = self.with_context(active_test=False)
+        return super(Route, self).write(values)
```

Comment:
nit: `super(Route, self)` -> `super()`

Reply:
The reason why i added with self is because i changed self and wanted to make it more visible that will be running as self.
In fact should change anything , but i think we have freedom on this part to use or not self as we are commonly using both ways in our code.

Reply:
yeah it won't change anything and I understand it makes it more visible, I just thought we had a rule where we didn't pass in `Route, self` when we didn't need to, as I have been asked to change this a lot in the past. Either way it shouldn't hold up the story as is only a nit :slightly_smiling_face:

Reply:
will resolve the thread

Reply:
Thanks, if you can add tag reviewed on the story i would appreciate it.